views/result/account.validation.pending.html

```diff
@@ -8,10 +8,17 @@ <h3 class="govuk-notification-banner__heading">
         <div class="gds-loading-icon"></div>
     </div>
 
-    <script>
-        console.log("Reloading in {{resultReloadDurationSeconds}} seconds")
-        setTimeout(() => location.reload(), {{resultReloadDurationSeconds}}000);
+    <script defer>
+        const eventSource = new EventSource("/xbrl_validate/result/{{fileId}}/sse");
```

Comment:
We have some constants for Url paths that should be used here.
The Urls are already passed to the template as globals so they should just be able to be used here.
```suggestion
        const eventSource = new EventSource("{{Urls.SUBMIT}}/{{fileId}}/sse");
```

Reply:
Thanks has now been resolved